src/components/Contacts.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import Contact from "./Contact";
+import contacts from "./../contacts.json";
+const Contacts = () => {
+  const data = contacts.slice(0, 5);
+
+  const [addContact, setAddContact] = useState(data);
+  const random = Math.floor(Math.random() * contacts.length);
+
```

Comment:
Aca podemos hacer una validacion para no tener contactos repetidos 👀 
```suggestion
const newContact = contacts[random];
  const generateRandomContact = () => {
    if (!addContact.includes(newContact)) {
    }
    setAddContact([...addContact, newContact]);
  };
```